dev-infrastructure/Makefile

```diff
@@ -40,26 +37,19 @@ rg: setsubscription
   		--location $(LOCATION) \
 		--tags "CreatedByConfig=${AKSCONFIG}"
 
-aks.linked: setsubscription rg
+dev.infrastructure: setsubscription rg
 	az deployment group create \
-		--name "$(DEPLOYMENTNAME)-aks" \
+		--name "$(DEPLOYMENTNAME)-dev-infra" \
 		--resource-group $(RESOURCEGROUP) \
 		--template-file $(AKSTEMPLATE) \
 		--confirm-with-what-if \
 		--parameters \
-			kubernetesVersion=$(AKSVERSION) \
-			vnetAddressPrefix=$(VNETCIDR) \
-			subnetPrefix=$(NODESUBNETCIDR) \
-			podSubnetPrefix=$(PODSUBNETCIDR) \
-			currentUserId=$(CURRENTUSER) \
-			enablePrivateCluster=$(PRIVATE_AKS) \
-			createdByConfigTag=${AKSCONFIG}
-
-
-aks.standalone: setsubscription rg aks.linked
+			configurations/${AKSCONFIG}.bicepparam \
+		--parameters \
+			currentUserId=$(CURRENTUSER)
 
 aks.kubeconfig:
```

Comment:
since the makefile is nested, we should update the .gitignore to be `*kubeconfig` instead of `/*kubeconfig`